.env.example

```
@@ -26,3 +26,10 @@ PRODUCT_SERVICE_SCHEMA=product_schema
 HUB_SERVICE_NAME=hub-service
 HUB_SERVICE_PORT=19200
 HUB_SERVICE_SCHEMA=hub_schema
+
+# api-gateway
+GATEWAY_NAME=api-gateway
+GATEWAY_PORT=19000
+
+# SECURITY
+JWT_SECRET_KEY=your_very_long_and_safe_random_string_64_bytes_recommend
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)